src/lib/isAlpha.js

```diff
@@ -1,8 +1,22 @@
 import assertString from './util/assertString';
 import { alpha } from './alpha';
 
-export default function isAlpha(str, locale = 'en-US') {
-  assertString(str);
+export default function isAlpha(_str, locale = 'en-US', options = {}) {
```

Comment:
I am just curious why change `str` to `_str` and later do 
```javascript 
let str = _str
```
I thought we could pass `str` as before and do: 
```javascript
assertString(str);
```
But, all in all, it looks good to me.

Reply:
@ezkemboi ah it's not good practice to directly change the arguments, just following best practices, let me know if can go ahead and resolve the conflict